fr.sgued.ten-forward.json

```diff
@@ -0,0 +1,46 @@
+{
+  "app-id": "fr.sgued.ten-forward",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "ten-forward",
+  "finish-args": [
+      "--env=RUST_LOG=info",
+      "--share=network",
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri"
+  ],
+  "modules": [{
+    "name": "ten-forward",
+    "buildsystem": "simple",
+    "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+      "env": {
+        "CARGO_HOME": "/run/build/ten-forward/cargo"
+      }
+    },
+    "build-commands": [
+      "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+      "cargo --offline build --release --verbose",
+      "install -Dm755 ./target/release/ten-forward -t /app/bin/"
+    ],
+    "post-install": [
+      "install -Dm755 dist/fr.sgued.ten-forward.metainfo.xml /app/share/appdata/fr.sgued.ten-forward.metainfo.xml",
+      "install -Dm755 dist/fr.sgued.ten-forward.desktop /app/share/applacations/fr.sgued.ten-forward.desktop",
```

Comment:
```
desktop-file-validate fr.sgued.ten-forward.desktop 
fr.sgued.ten-forward.desktop: error: first group is not "Desktop Entry"
fr.sgued.ten-forward.desktop: error: file contains group "Desktop entry", but groups extending the format should start with "X-"
```

This makes appstream-compose fail

Reply:
It shows up at the very end of flatpak-builder output.

Reply:
I saw it initially for some errors with the metainfo file but it didn't show up for the desktop entry because of the type in the path that you noticed.

Reply:
I think it's fixed now. But then running manually `desktop-file-validate` on the entry raised some issues with categories that where not in the flatpak-builder output. I fixed them too.